sbpy/units/core.py

```diff
@@ -133,3 +136,150 @@ def spectral_density_vega(wfb):
         (flam0.unit, VEGA, lambda x: x / flam0.value,
          lambda x: x * flam0.value)
     ]
+
+
+@u.quantity_input(cross_section='km2', reflectance='1/sr',
+        f_sun=['W/(m2 um)', 'W/(m2 Hz)'])
+def reflectance(cross_section=None, reflectance=None, wfb=None, M_sun=None,
+        f_sun=None):
+    """Reflectance related equivalencies.
+
+    Support conversion from/to reflectance and scattering cross-section
+    to/from total flux and magnitude.
+
+    The default magnitude system is ``VEGAmag``, where the apparent magnitude
+    of Vega is assumed to be 0 in all and any wavelengths and bands.  If other
+    magnitude system is used, then it is implicitly inferred from the solar
+    magnitude passed by keyword 'M_sun'.
+
+    Parameters
+    ----------
+    cross_section : `astropy.units.Qauntity`
+        Total scattering cross-section
+    reflectance : `astropy.units.Quantity`
+        Average reflectance
```

Comment:
One of `cross_section` or `reflectance` is required, right?

Reply:
Instead, can the equivalencies be expressed in per unit area and per unit reflectance so that the user does not need to add them to the `reflectance` call?  I guess in that case `rh` and `delta` are needed, right?

Reply:
Well, we can express the scattering cross-section in terms of solid angle, and that's what you actually mean by "per unit area", right?  I think this is kind of overdoing (or maybe underdoing) and requires one extra step for users to understand this equivalency, and therefore defeats the purpose of providing convenience to users.

I debated about adding `rh` and `delta` into the equivalence call, and decided not, because then we'll also need to add a phase function model to the parameter list.  That's going to be too much of integration and will make this equivalency too complicated.  I think we want to keep it clean and simple.

Reply:
Sounds good.